packages/demo/package.json

```diff
@@ -0,0 +1,67 @@
+{
+    "name": "react-vapor-demo",
+    "version": "5.7.5",
+    "description": "Vapor Design System",
+    "main": "src/Index.ts",
+    "scripts": {
+        "start": "webpack-dev-server",
+        "build": "webpack",
+        "precommit": "lint-staged"
+    },
+    "author": "Coveo",
+    "license": "Apache-2.0",
+    "homepage": "https://react-vapor.prod.cloud.coveo.com/",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/coveo/react-vapor.git"
+    },
+    "dependencies": {
+        "codemirror": "^5.51.0",
```

Comment:
You have a funny combinaison of `^version` and `version` in both dependencies and devDependencies